### PR TITLE
Integration testing

### PR DESCRIPTION
**Pull request by RaneWallin**

### CODE REVIEW

**Comment by RaneWallin** [comment]
[Configure integration testing](https://trello.com/c/vaeGoRDj/6-configure-integration-testing)

File: socialtours/src/components/TestComponent.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+const TestComponent = _ => {
+    const [h1Message, setH1Message] = useState('default');
```
Comment (bcuz) [inline socialtours/src/components/TestComponent.js]: hooks, dope

**Comment by bcuz** [comment]
this PR is helpful. the trello card you linked to says to set up something called Cypress.io, but react-testing-library shouldve probably been implemented as a part of [this card](https://trello.com/c/Agmy5Inr/19-set-up-test-suite) (but wasn't)

**Comment by RaneWallin** [comment]
I added cypress integration to this PR for both integration and E2E testing

**Pull request merged**